### PR TITLE
Het ambacht

### PR DESCRIPTION
**Pull request by DorinevanderVlies**

### CODE REVIEW

File: db/schema.rb
Comment (kndehaan) [inline db/schema.rb:1]: Why is this changed?

Comment (DorinevanderVlies) [reply db/schema.rb:1]: Sorry, I commited that accidentally. Thanks for spotting it!
